.github/workflows/weekly-sync-from-snapshot-mainnet-paritydb-pruned.yml

```diff
@@ -0,0 +1,23 @@
+---
+# This workflow performs sync to Mainnet from a pruned ParityDB snapshot using the latest
+# main version.
```

Comment:
This is annoyingly incorrect – we expect syncs from the latest main to fail, since mainnet can be a version behind and we don't keep compatibility with the older versions. If you want this to actually conform to our assumptions you should be using the latest testnet release branch instead. I don't know how much work it would be to fix this though, and this PR is doing so many good things that I would prefer not delaying it too much. At the same time we are currently afaik not planning to break compatibility with whatever is on mainnet even after release 14, so we would have some time to fix this before it starts consistently failing, so maybe creating a task and leaving a comment about this possibly failing would be enough.

Reply:
Good point, you're right this mainnet test as it is now how major logic flow. I'll fix in the next PR which I'm going to add anyway, altogether with support for two additional tests (Mainnet/Testnet RocksDB sync test). The fix simply would be to run Mainnet sync Tests from Testnet binary.